FWCore/Framework/src/DataProxy.cc

```diff
@@ -102,7 +103,7 @@ namespace edm {
         auto waitTaskPtr = waitTask.get();
         auto token = ServiceRegistry::instance().presentToken();
         edm::esTaskArena().execute([this, waitTaskPtr, &iRecord, &iKey, iEventSetupImpl, token]() {
-          prefetchAsync(waitTaskPtr, iRecord, iKey, iEventSetupImpl, token);
+          { prefetchAsync(WaitingTaskHolder(waitTaskPtr), iRecord, iKey, iEventSetupImpl, token); }
```

Comment:
Why the extra braces?

Reply:
I think in an earlier version of my edit I had a WaitingTaskHolder variable so the scope was needed. Now they are just a left over.